rubin_jupyter_utils/hub/scanrepo/scanrepo.py

```diff
@@ -544,16 +526,10 @@ def _reduce_results(self):
                 ict = imap[ikey]["count"]
                 if ict:
                     r[ikey] = displayorder[idx][:ict]
-            all_tags = self._sort_tags_by_date()
-            self._all_tags = all_tags
-            self.data = r
 
-    def _sort_tags_by_date(self):
-        items = [x[1] for x in self._results_map.items()]
-        dec = [(x["last_updated"], x["name"]) for x in items]
-        dec.sort(reverse=True)
-        tags = [x[1] for x in dec]
-        return tags
+            self._all_tags = [x[1]['name'] for x in self._results_map.items()]
+            self._all_tags.reverse()
```

Comment:
Um, maybe if you leave this out and change the index, that fixes my complaint about display ordering?  Or maybe you changed it while I was reviewing and it didn't used to be this way.  Or....where did the sort go?  OK, I'm lost now.

Reply:
I changed it.